src/scripts/components/header/ContentDropdown.js

```diff
@@ -49,6 +49,20 @@ class ContentDropdown extends React.Component {
     this.context.executeAction(AyahsActions.updateAyahs, {
       content: chosenOptions
     });
+
+    if(chosenOptions.toString() === ''){
```

Comment:
Let's keep the coding style the same. Spaces between `if` and `(`, so `if (` and the curl `) {`.

Thanks!